tests/unit/model.py

```diff
@@ -23,7 +23,6 @@
 
 from haas.model import *
 from haas import config
-from haas.ext.obm.ipmi import Ipmi
```

Comment:
I suppose I could write a subclass of ModelTest that does the import and attaches it to self (and then have the tests that use ipmi inherit from it), or rig up a pytest fixture, but neither of these actually save any code (both are longer than just doing the import in each place), and I think they obscure what's going on. Possible implementations:

```python
# fixture version. Also need to add this fixture to each of the tests that use it.
@pytest.fixture()
def Ipmi():
    from haas.ext.obm import Ipmi as ret
    return ret
```

```
# class version. Each test that uses this now needs to inherit from it, instead of ModelTest
class IpmiModelTest(ModelTest):
    def setUp(self):
        from haas.ext.obm.ipmi import Ipmi
        self.Ipmi = Ipmi
```

My inclination is to just do what we're doing now; I can't think of a way to dedup this that I think is actually better than doing the import multiple times. @knikolla, thoughts?